### ISSUE TITLE
ASM not intialized (136), Startup crash (138)

### ISSUE THREAD

**Issue by walkerjonny** [Darkhax-Minecraft/Bookshelf#19]
Hi,
I'm getting a huge amount of those lines blowing up my latest.log (> 1 GB):
"The ASM has not been initialized, there is an error with your setup!"
At the moment I have added Bookshelf-1.7.10-1.0.3.136.jar.
This happens at world loading and continues then.
I have removed Enchanting Plus 4.0 beta temporarily because of an error I reported there.

The newest Bookshelf-1.7.10-1.0.3.138.jar caused a startup crash for me as I tried it.
[latest.log.txt](https://github.com/Darkhax-Minecraft/Bookshelf/files/49348/latest.log.txt)

Can you have a look into this, please?

**Comment by darkhax** [comment]
The ASM not being initialized was fixed in build 138. The crash you are linking for 138 is related to the SAPManPack mod. I believe there is an incompatibility between Bookshelf and that mod. The SAPManPack mod is actually being merged into Bookshelf right now, so we won't be releasing a compatibility patch.

**Comment by walkerjonny** [comment]
So I should remove SAPManPack and all mods that require it, right?
Will those mods be updated to the bookshelf api?

**Comment by darkhax** [comment]
Yes, removing the SAPManPack and all related mods will solve this issue. Keep in mind that E+ is just in Beta phases right now, so once you're done testing you can uninstall it and use the SAPManPack mods again. The author of SAPManPack has told me that he intends to port all of his mods over to Bookshelf once our merger is complete.

**Comment by walkerjonny** [comment]
Thank you! 
Good to know.
